vb-be/src/routes/playlist.js

```diff
@@ -0,0 +1,73 @@
+"use strict";
+
+const { spotifyApi } = require("../lib/spotify");
+const express = require("express");
```

Comment:
just a nit, but let's organize the imports by external modules, then imports.  I'll see if I can set up some import paths for us soon so we can do like we do on the FE

Reply:
Oh the eslint isn't set up for the backend. The backend is pretty shaky rn lol

Reply:
I mean eslint is there but it does very less

Reply:
What do you think we should add to the linter for the BE?  We can arrange that soon though, but we should make sure our imports etc are still being implemented right even without the linter